File: AuraFit-BackEnd/src/main/java/app/aurafitbackend/DTOS/Cart_And_Orders_DTOS/CheckoutRequestDTO.java
package app.aurafitbackend.DTOS.Cart_And_Orders_DTOS;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CheckoutRequestDTO {

    private String firstName;
    private String lastName;
    private String email;
    private String phone;
    private String address;
    private String city;
    private String postalCode;

}

File: aurafit-frontend/src/Models/DTOS/CheckoutRequestDTO.ts


export class CheckoutRequestDTO {
    firstName: string;
    lastName: string;
    email: string;
    phone: string;
    address: string;
    city: string;
    postalCode: string;

    constructor(firstName: string, lastName: string, email: string, phone: string, address: string, city: string, postalCode: string) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.email = email;
        this.phone = phone;
        this.address = address;
        this.city = city;
        this.postalCode = postalCode;
    }


}